backend/app/controllers/rooms.controller.js

```diff
@@ -65,4 +65,40 @@ const fetchRooms = (strategy) => {
   }
 };
 
-export default fetchRooms;
+const create = (req, res) => {
+  const newRoom = {
+    id: req.query.roomId,
+    name: req.query.roomName,
+    disableMeeting: false,
+    temporary: true,
+  };
+
+  const found = req.app.locals.roomsDetail.find(
+    element => element.id == req.query.roomId,
+  );
+
+  if (!found) {
+    req.app.locals.roomsDetail.splice(1, 0, newRoom);
+  }
```

Comment:
I think that is a great time to move rooms creation / fetch logic to a proper service or store interface.

That will allow us to create different rooms store strategies, like a backoffice endpoint.
What you think?

Reply:
I totally agree!

That's inclusive my next PR that is ready waiting by this context "organization". =)